Updated PhysX Wind Provider topic. Updated steps moved/replaced images. (#1473)
o3de/o3de.org: content/docs/user-guide/components/reference/physx/cloth.md

## Wind properties 

{{< note >}}

> The component wind properties create wind that affects only the cloth referenced by the component. To create wind that can affect multiple components across multiple entities, see [Create Wind Forces](/docs/learning-guide/tutorials/physx/wind-provider).

Reviewer: ```suggestion
The component wind properties create wind that affects only the cloth that the component references. For instructions to create wind that can affect multiple components across multiple entities, refer to [Create Wind Forces](/docs/learning-guide/tutorials/physx/wind-provider).
```
